tests/basic/storage_pool_test.go

```diff
@@ -12080,3 +12081,234 @@ var _ = Describe("{PoolDeleteMultiplePools}", func() {
+				})
+				log.FailOnError(err, "error while enabling trashcan")
+				log.InfoD("Trashcan is successfully enabled")
+			})
+
+		stepLog = "Schedule Applications on the cluster and get details of Volumes"
+		Step(stepLog, func() {
+			for i := 0; i < Inst().GlobalScaleFactor; i++ {
+				contexts = append(contexts, ScheduleApplications(fmt.Sprintf("volresizeallvol-%d", i))...)
+			}
+		})
+		appsDeleted := false
+		deleteApps := func() {
+			if !appsDeleted {
+				appsValidateAndDestroy(contexts)
+			}
+		}
+
+		ValidateApplications(contexts)
+		defer deleteApps()
+
+		stepLog = "Verify parallel resize of all the volumes "
+		Step(stepLog, func() {
+			for _, eachCtx := range contexts {
+				vols, err := Inst().S.GetVolumes(eachCtx)
+				log.FailOnError(err, "Failed to get list of Volumes in the cluster")
+				for _, eachVol := range vols {
+					volDetails := VolumeDetails{}
+
+					// Get details on the Volume
+					volDetails.vol = eachVol
+
+					// Get PVC object from the Volume
+					pvc, err := GetPVCObjFromVol(eachVol)
+					log.FailOnError(err, "Failed to get PVC Details from Volume [%v]", eachVol.Name)
+
+					// Update volSizeMap to perform Parallel Resize of all the Volumes present in the cluster
+					volDetails.pvc = pvc
+					volSizeMap = append(volSizeMap, &volDetails)
+				}
+			}
+		})
+
+		for _, eachVol := range volSizeMap {
+			// Pod details after blocking IP
+			podsAfterblk, err := k8sCore.GetPodsUsingPVC(eachVol.vol.Name, eachVol.vol.Namespace)
+			log.FailOnError(err, "unable to find the node from the pod")
+			for _, eachPodAfter := range podsAfterblk {
+				log.Infof(fmt.Sprintf("Current state of the Pod [%v] is [%v]", eachPodAfter.Name, eachPodAfter.Status.Phase))
+				if eachPodAfter.Status.Phase != "Running" {
+					log.Infof(fmt.Sprintf("State of the Pod before Resize volume [%v] is [%v]", eachPodAfter.Name, eachPodAfter.Status.Phase))
+					log.FailOnError(fmt.Errorf("Pod [%v] Consuming Volume [%v] is not in Running State ",
+						eachPodAfter.Name, eachVol.vol.Name), "Pod not in Running State")
+				}
+			}
+		}
+
+		// Volume resize routine resizes specific Volume
+		log.InfoD("start Volume resize on each Volume present in the cluster")
+		ResizePvcInParallel := func(newVolumeIDs *VolumeDetails) {
+			defer GinkgoRecover()
+			volCurSize := newVolumeIDs.vol.Size / units.GiB
+			volNewSize := volCurSize + uint64(10)
+			log.Infof("Resizing Volume [%v] from size [%v] to [%v]", newVolumeIDs.vol.Name, newVolumeIDs.vol.Size, volNewSize*units.GiB)
+			vol, err := Inst().S.ResizePVC(newVolumeIDs.ctx, newVolumeIDs.pvc, volNewSize)
+			log.FailOnError(err, "Failed to resize PVC [%v]", vol.Name)
+
+			time.Sleep(60 * time.Second)
+			volReSize, err := Inst().V.InspectVolume(vol.ID)
+			log.FailOnError(err, "inspect returned error ?")
+
+			log.Infof("Verify volume size after resizing the volume [%v]", vol.Name)
+			dash.VerifyFatal(volReSize.Spec.Size > newVolumeIDs.vol.Size, true, "Resize of Volume didnot happen!")
```

Comment:
Can we also add check for the new size apart from simply checking less than and greater than.